code/modules/paperwork/paper.dm

```diff
@@ -437,3 +437,52 @@
+
+/obj/item/weapon/paper/oldstat/protosuit
+	name = "B01-RIG Hardsuit Report"
+	info = "<b>*Prototype Hardsuit*</b><br><br>The B01-RIG Hardsuit is a prototype powered exoskeleton. Based off of a recovered pre-void war era united earth government powered military \
+	 exosuit, the RIG Hardsuit is a breakthrough in Hardsuit technology, and is the first post-void war era Hardsuit that can be safely used by a operator.<br><br>The B01 however suffers \
+	  a myriad of constraints. It is slow and bulky to move around, it lacks any significant armor plating against direct attacks and its internal heads up display is unfinished,  \
+	  resulting in the user being unable to see long distances.<br><br>The B01 is unlikely to see any form of mass production, but will serve as a base for future Hardsuit developments."
+
+/obj/item/weapon/paper/oldstat/protohealth
+	name = "Health Analyser Report"
+	info = "<b>*Health Analyser*</b><br><br>The portable Health Analyser is essentially a handheld varient of a health analyser. Years of research have concluded with this device which is \
+	capable of diagnosing even the most critical, obscure or technical injuries any humanoid entity is suffering in a easy to understand format that even a non-trained health professional \
+	 can understand.<br><br>The health analyser is expected to go into full production as standard issue medical kit."
+
+/obj/item/weapon/paper/oldstat/protogun
+	name = "K14 Energy Gun Report"
+	info = "<b>*K14-Multiphase Energy Gun*</b><br><br>The K14 Prototype Energy Gun is the first Energy Rifle that has been successfully been able to not only hold a larger ammo charge \
+	than other gun models, but is capable of swapping between different energy projectile types on command with no incidents.<br><br>The weapon still suffers several drawbacks, its alternative, \
+	non laser fire mode, can only fire one round before exhausting the energy cell, the weapon also remains prohibitively expensive, nonetheless NT Market Research fully believe this weapon \
+	will form the backbone of our Energy weapon cataloge.<br><br>The K14 is expected to undergo revision to fix the ammo issues, the K15 is expected to replace the 'stun' setting with a \
+	'disable' setting in a attempt to bypass the ammo issues."
+
+/obj/item/weapon/paper/oldstat/protosing
+	name = "Singularity Generator"
+	info = "<b>*Singularity Generator*</b><br><br>Modern power generation typically comes in two forms, a Fusion Generator or a Fission Generator. Fusion provides the best space to power \
+	ratio, and is typically seen on military or high security ships and stations, however Fission reactors consume massive amounts of resources to keep operational, and are highly prone to \
```

Comment:
Space Corps are so cheap they create knock off Fission Reactors. I dunno its just a fluff piece.

I'll reword it that Fission Reactors require very rare and expensive construction materials to make, hence why they are rare.

There's a reason the Centcom Ferry can reach SS13 within ten seconds, yet the evac ship needs three minutes.